src/components/v2/ProgressBar/styles.ts

```diff
@@ -1,22 +1,33 @@
 import { css } from '@emotion/react';
 import { useTheme } from '@mui/material';
 
-export const useStyles = ({ over, secondaryOver }: { over: boolean; secondaryOver: boolean }) => {
+export const useStyles = ({
+  over,
+  secondaryOver,
+  progressColorOverride,
+}: {
+  over: boolean;
+  secondaryOver: boolean;
+  progressColorOverride?: string;
+}) => {
   const theme = useTheme();
   return {
     slider: css`
       display: block;
       color: ${over ? theme.palette.interactive.error50 : theme.palette.interactive.success};
+      color: ${progressColorOverride};
```

Comment:
We could also use a  success color prop with a default theme.palette.interactive.success. This makes the component more flexible, keeps the overflow logic, and removes the duplicate color attribute declaration 
```
color: ${over ? theme.palette.interactive.error50 : successColor};
```